spec/oga/xml/element_spec.rb

```diff
@@ -161,6 +165,22 @@
       @element.get('class').should == 'foo'
     end
 
+    it 'adds a new attribute' do
+      @element['class'] = 'foo'
+
+      @element['class'].should == 'foo'
+    end
+
+    it 'converts symbols to strings in names' do
+      @element[:foo] = 'foo'
+      @element[:foo].should == 'foo'
+      @element['foo'].should == 'foo'
+
+      @element['bar'] = 'bar'
+      @element[:bar].should == 'bar'
+      @element['bar'].should == 'bar'
+    end
```

Comment:
This spec should be scoped to a `describe '#[]='` outside of the `describe '#set'`.